registry/client/client.go

```diff
@@ -261,3 +265,26 @@ func normalizeConfig(config *Config) error {
 
 	return nil
 }
+
+// Debug is used to set the debugging flag.
+func (client *Client) Debug(debug bool) {
+	client.debug = debug
+}
+
+// printCurl prints the curl of a given request
+func (client *Client) printCurl(method, url string, data interface{}, headers http.Header) {
```

Comment:
Pretty sure there is at least one other printCurl() function declaration so perhaps this should be moved to a util function